modules/claims_api/app/sidekiq/claims_api/v2/poa_form_builder_job.rb

```diff
@@ -65,10 +58,24 @@ def data(power_of_attorney, form_number, rep)
                           'firstName' => rep.first_name,
                           'lastName' => rep.last_name
                         } })
+
+        res.deep_merge!(organization_name(power_of_attorney)) if form_number == '2122'
```

Comment:
Organization name only needs to appear on the 2122 form for now. If we ever support `SERVICE ORGANIZATION REPRESENTATIVE` on 2122a, then we'll need to add it.